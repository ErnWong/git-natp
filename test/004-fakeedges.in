
     ..         .--D-_
       \      .'      \
    ----A----B---    --E------F H --G----
         '._  \       /            /
                      \___________/
