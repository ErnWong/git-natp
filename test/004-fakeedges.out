A
B A
D B
E D
F E
G
[H] F
